buying_selling/users/api/serializers.py

```diff
@@ -53,6 +54,7 @@ class Meta:
 class ProfileUpdateSerializer(ModelSerializer):
 
     user = UserDetailSerializer()
+    hostel = CharField(source='hostel.name', read_only=True)
```

Comment:
We don't need hostel name in update endpoint.

Reply:
Yeah, this was a bug. Fixed it!